### DISCUSSION TITLE
Window state on iPadOS

### DISCUSSION THREAD

**Discussion by Tunous** [community/community#15035] [Mobile]
Currently GitHub app windows don't remember their state on iPad. If I leave a window, where I navigated to some screen, and then return to it sometime later I am reverted back to start screen if system decided to suspend the app.

I would like for the app to remember my position similarity to how it is done in Safari.

**Comment by stevepopovich** [comment]
Hi there! I passed this on to the team for prioritization. Thanks for the feedback!
